From aa8b1266f86fce097627d48545b799091b22f7f9 Mon Sep 17 00:00:00 2001
From: Angel Hitov <ahitov@codeaurora.org>
Date: Mon, 11 Mar 2019 12:42:20 -0700
Subject: [PATCH 1/3] Add insecure flag to curl command

This is a workaround for native curl missing certificate in LE.
---
 tensorflow/lite/tools/make/download_dependencies.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tensorflow/lite/tools/make/download_dependencies.sh b/tensorflow/lite/tools/make/download_dependencies.sh
index fa3d5d3..a263049 100755
--- a/tensorflow/lite/tools/make/download_dependencies.sh
+++ b/tensorflow/lite/tools/make/download_dependencies.sh
@@ -61,12 +61,12 @@ download_and_extract() {
   echo "downloading ${url}" >&2
   mkdir -p "${dir}"
   if [[ "${url}" == *gz ]]; then
-    curl -Ls "${url}" | tar -C "${dir}" --strip-components=1 -xz
+    curl --insecure -Ls "${url}" | tar -C "${dir}" --strip-components=1 -xz
   elif [[ "${url}" == *zip ]]; then
     tempdir=$(mktemp -d)
     tempdir2=$(mktemp -d)

-    curl -L ${url} > ${tempdir}/zipped.zip
+    curl --insecure -L ${url} > ${tempdir}/zipped.zip
     unzip ${tempdir}/zipped.zip -d ${tempdir2}

     # If the zip file contains nested directories, extract the files from the
--
1.9.1

